core/libp2p/connection/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+#
+# Copyright Soramitsu Co., Ltd. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+add_subdirectory(security_conn_impl)
```

Comment:
better make dir `/core/libp2p/connection/plaintext` and header `/core/libp2p/connection/plaintext.hpp`